app/helpers/view_hook_helper.rb

```diff
@@ -0,0 +1,13 @@
+module ViewHookHelper
+
+  # Look up the current path and render the view hooks for a specific placement
+  def view_hook(placement, args = {})
```

Comment:
When I'm looking through some haml, the word "render" sticks out to flag something external. I like how `#view_hook` as a name works as a companion to `#add_hook`, but something like `#render_view_hook` might have a more clear association for me.